src/server/active_tran_server.cpp

```diff
@@ -251,6 +251,21 @@ active_tran_server::get_log_page_broker ()
   return *m_log_page_broker;
 }
 
+void active_tran_server::init_has_remote_storage (const SERVER_TYPE a_server_type)
+{
+  // precondition: server type must have been initialized
+  assert_is_active_tran_server ();
+
+  m_has_remote_storage = prm_get_bool_value (PRM_ID_REMOTE_STORAGE);
+}
```

Comment:
m_has_remote_storage initialization can be included in init_page_server_hosts(), because it determines the outcome of page server connections initialization. Later on it will become always true.

Reply:
Done